src/components/radio-button/_radio-button.scss

```
@@ -230,6 +235,61 @@
   .#{$prefix}--radio-button__label.#{$prefix}--skeleton .#{$prefix}--radio-button__appearance {
     display: none;
   }
+
+  .#{$prefix}--radio-button-wrapper .#{$prefix}--radio-button__label {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 0px;
```

tiny nitpick but just a small inconsistency with the other styles. on a side note, we can configure prettier to handle this if we haven't already

```suggestion
    margin: 0;
```